calculator.js

```diff
@@ -0,0 +1,123 @@
+// Get the user inputs
+
+var doMath = prompt("Type anything to do math!");
+if(doMath)
+{
+  for (var i = 1; i >= 0; i++)
+  {
+    console.log("-----");
+    calcChoice = prompt("Which calculator do you want to use?  (B)asic | (A)dvanced | (BMI)")
+    if(calcChoice == "b")
+    {
+      var value1 = parseInt(prompt('enter first value'));
+      var value2 = parseInt(prompt('enter the second value'));
+      var choice = prompt('Choose your action (a)dd (s)ubtract (m)ultiply (d)ivide (Mo)dulus') || "a";
+
+      if(choice == "a")
+      {
```

Comment:
Writing out code blocks like this is fairly difficult to read, I would stick to the standard way.